backup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mysqldump=/usr/bin/mysqldump
+mysql_user=root
+mysql_password=root
+
+now=$(date +"%d_%m_%Y")
+
+docker exec -i lmnad_mysql_dev $mysqldump --user=$mysql_user  --password=$mysql_password --databases lmnad_db | gzip > backup/backup_$now.sql.gz
+
```

Comment:
Please, add `docker cp`  command for copying sql.gz file from container to host, set the target directory as a variable at beginning of the script. Thank you.